fem/fespace.hpp

```diff
@@ -983,6 +983,14 @@ class QuadratureInterpolator
    void Mult(const Vector &e_vec, unsigned eval_flags,
              Vector &q_val, Vector &q_der, Vector &q_det) const;
 
+   /// Interpolate the values of the E-vector @a e_vec at quadrature points.
+   /// This computation uses tensor product structure.
+   void Values(const Vector &e_vec, Vector &q_val) const;
+
+   /// Interpolate the derivatives of the E-vector @a e_vec at quadrature points.
+   /// This computation uses tensor product structure.
+   void Derivatives(const Vector &e_vec, Vector &q_der) const;
+
```

Comment:
It looks like this computes derivatives with respect to reference space. Could we add a method that maps it back to physical space? I think this is generic enough that we should be able to calculate derivatives at dofs (as I need) or quadrature points.

Reply:
`done` via PR #1270 , thanks @artv3